ServerCore/Pages/Resources/PH20/FAQPartial.cshtml

```diff
@@ -0,0 +1,98 @@
+﻿<div class="container">
+    <h1>FAQ</h1>
```

Comment:
I'm going to assume that you copied this from somewhere and not review the actual text. If you'd like me to actually review the content lmk